src/Microsoft.AspNetCore.Mvc.ApiExplorer/exceptions.net45.json

```diff
@@ -0,0 +1,35 @@
+﻿[
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescriptionProviderContext",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiParameterDescription",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiParameterRouteInfo",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiRequestFormat",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiResponseFormat",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public class Microsoft.AspNetCore.Mvc.ApiExplorer.ApiResponseType",
+        "Kind": "Removal"
+  },
+  {
+        "OldTypeId": "public interface Microsoft.AspNetCore.Mvc.ApiExplorer.IApiDescriptionProvider",
+        "Kind": "Removal"
+  },  
```

Comment:
Remove the trailing comma to make this file valid. Also remove the extra blank line just below.

Same for the other exclusions file.